lib/parser/lexer/literal.rb

```diff
@@ -227,6 +227,17 @@ def extend_space(ts, te)
       end
     end
 
+    def supports_line_continuation_via_slash?
+      heredoc? ||
+        regexp? ||
+        (@str_type == '"') ||
+        (@str_type == "%{") ||
+        (@str_type == "%Q{") ||
+        (@str_type == ':"') ||
+        (@str_type == "%x{") ||
+        (@str_type == '`')
```

Comment:
Perhaps it makes more sense to list the types that don't support this, which are the word and symbol lists.

Reply:
Good point!